lib/commands.coffee

```diff
@@ -50,7 +57,7 @@ class Commands
   lint: ->
     try
       @linter.getActiveEditorLinter()?.lint(false)
-      @linter.views.render()
+      @linter.views.render(@linter.messages.publicMessages)
```

Comment:
This line was actually redundant and no longer required after the EditorLinter isolation, gonna remove. Also we have a lot of weird accesses, I tried to remove them in `linter-ng`, porting those changes here now.